xrpl/models/currencies/xrp.py

```diff
@@ -77,3 +77,12 @@ def to_amount(self: XRP, value: Union[str, int, float]) -> str:
         if isinstance(value, str):
             return xrp_to_drops(float(value))
         return xrp_to_drops(value)
+
+    def __repr__(self: XRP) -> str:
+        """
+        Generate string representation of XRP.
+
+        Returns:
+            A string representation of XRP currency.
+        """
+        return "XRP()"
```

Comment:
Why does the XRP currency have brackets?

Reply:
It has parentheses so it matches the other `__repr__` outputs - e.g. issued currencies are `IssuedCurrency(currency="...", issuer="r...")`